### PR TITLE
Adds customer success manager role

### PR DESCRIPTION
**Pull request by gabrielamatias**
@jacobpgn , @jesssummerfield : please review.

### CODE REVIEW

File: app/pages/about/jobs/positions/customer-success-manager.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            You believe in making sure your clients are at the forefront of our efforts.
+          </li>
+        </ul>
+        <p className='para'>Not essential, but nice to have:</p>
+        <ul className='list'>
+          <li>
+            Previous relevant experience in a similar role.
+          </li>
+          <li>
+            Previous experience in Customer Success in a B2B SaaS.
```
Comment (jacobpgn) [inline app/pages/about/jobs/positions/customer-success-manager.js]: Might just be me, but this sounds odd. `Previous experience in Customer Success for B2B SaaS`, perhaps?

**Comment by jacobpgn** [comment]
Just one comment, otherwise :+1:

**Comment by gabrielamatias** [comment]
@jacobpgn : what about now?

**Comment by jacobpgn** [comment]
Hmmmmm, still seeing the old version!

**Comment by gabrielamatias** [comment]
try now.

**Pull request merged**